README.rst

```diff
@@ -115,8 +115,10 @@ Changelog
 
 Version 2.0.0 (unreleased)
 --------------------------
-* Added support for OpenSlides 2.x.
+* Added support for OpenSlides 2.1.x.
 * Added support for speaker list.
+* Added support for assignment polls.
+* Added possibility to change the seat number.
```

Comment:
Suggestion for the 2.0.0 section:
* Added support for OpenSlides 2.1.x with autoupdate for all incoming votes.
* New speakers voting (add/remove from list of speakers).
* New election voting (Yes/No/Abstain for one candidate or 1 of n candidates via keypad keys 1..10) with anonymize votes
* Updated motion voting, with anonymize votes.
* Added possibility to change the seat label.